web/src/stream_settings_ui.js

```diff
@@ -237,18 +235,6 @@ export function add_sub_to_table(sub) {
         // ID isn't known yet.  These are appended to the top of the
```

Comment:
I'm wondering about "because the stream ID isn't known yet" in the comment here. It was added in https://github.com/zulip/zulip/commit/a0d33fa1cf0ffa025463e72e8845a2b0583a2df5

`stream_id` is available here but I think that wouldn't help to determine if "stream was just created in this browser window".

Reply:
Yeah, we have the `stream_id` in the event, but the `stream_create` UI code doesn't have a value to compare it with.